stack-nightly.yaml

```diff
@@ -31,7 +32,7 @@ extra-deps:
 
 flags:
   # https://github.com/haskell/directory/issues/176
-  directory: {os-string: true}
+  directory: { os-string: true }
```

Comment:
Huh, I wonder why Restyled didn't get me when I... oh, I think we need to migrate this repo to `.github/workflows/restyled.yml` still.